src/main/java/creativei/manager/impl/ReminderManagerImpl.java

```diff
@@ -38,4 +42,25 @@ public ResponseObject create(ReminderVo reminderVo) {
             return ResponseObject.getResponse(ExceptionType.GENERAL_ERROR.getMessage(),ExceptionType.GENERAL_ERROR.getCode());
         }
     }
+
+    @Override
+    public ResponseObject getReminderByDateRange(ReminderDateVo reminderDateVo) {
+        logger.info("Get Reminder by date range Method");
+        List<Reminder> reminders= null;
+        Date fromDate=null;
+        Date  toDate=null;
+        try {
+            fromDate=LocalizationUtil.stringtoDateWithTimeConverter(reminderDateVo.getFromDate());
+            toDate=LocalizationUtil.stringtoDateWithTimeConverter(reminderDateVo.getToDate());
+            reminders = reminderService.getReminderByDateRange(fromDate,toDate);
+            List<ReminderVo> reminderVos=ResponseHelper.getReminderByDateRangeResponse(reminders);
+            return ResponseObject.getResponse(reminderVos);
+        } catch (ParseException e) {
+            logger.error(e.getMessage(),e);
```

Comment:
Also log values of toDate and fromDate in case of error

Reply:
In case of parse exception value of date is there within the exception.

Reply:
Add general exception block as well to cover any unchecked exception scenario. Log the exception and request values there.